client/my-sites/people/people-invites/index.jsx

```diff
@@ -37,14 +37,7 @@ class PeopleInvites extends React.PureComponent {
 
 		return (
 			<Card key={ invite.invite_key }>
-				Invited
-				{ ' ' }
-				<Gravatar user={ gravatarUser } />
-				{ ' ' }
-				<strong>{ userNameOrEmail }</strong>
-				{ ' ' }
-				as
-				{ ' ' }
+				Invited <Gravatar user={ gravatarUser } /> <strong>{ userNameOrEmail }</strong> as{' '}
```

Comment:
I'm not sure why this change was made but it might be worth reverting — the whitespace from before actually could be significant.

Reply:
Pretty weird because I did not touch this file, it might be some automatic update from Prettify or some other tool. I reverted the change.

Reply:
This change showed back up. I think it's happening when you're merging master back in or something — that "touches" this file, and so Prettier auto-formats it.

Reply:
Prettier automatically re-formats the file, I guess when `/** @format */` was added 7 days ago to the file the pre-commit hook that formats the file was not trigger so the file remained not formatted.

The output looks the same https://cloudup.com/cbpRPfab9Q0 vs https://cloudup.com/cXRtcphgsWm so I guess we don't really need to revert it.